src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -124,17 +98,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('read_enabled', help='Indicates whether read operation is allowed.', arg_type=get_three_state_flag())
         c.argument('write_enabled', help='Indicates whether write or delete operation is allowed.', arg_type=get_three_state_flag())
 
-    with self.argument_context('acr repository delete') as c:
-        c.argument('manifest', nargs='?', required=False, const='', default=None, help=argparse.SUPPRESS)
-        c.argument('tag', help=argparse.SUPPRESS)
-
     with self.argument_context('acr repository untag') as c:
         c.argument('image', arg_type=image_by_tag_type)
 
     with self.argument_context('acr create') as c:
         c.argument('registry_name', completer=None)
-        c.argument('deployment_name', arg_type=deployment_name_type, validator=None)
-        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('deployment_name', validator=None)
+        c.argument('location', validator=get_default_location_from_resource_group)
```

Comment:
Why did you remove these?  These still seem quite applicable.

Reply:
As requested in previous PRs, they are already registered as global params https://github.com/Azure/azure-cli/blob/f4a60fb3d0ab05ef9cbbb54e2999e80206d0d3a9/src/azure-cli-core/azure/cli/core/__init__.py#L250